ann_benchmarks/main.py

```diff
@@ -122,6 +123,12 @@ def main():
         type=positive_int,
         help='Number of Docker containers in parallel',
         default=1)
+    parser.add_argument(
+        '--batch-size',
+        help='Number of vectors in each batch. only works in batch mode. default (-1) is a single batch',
+        required=True,
```

Comment:
Can you make this not required? All the tests are failing because of this right now. Thanks!